src/Sandbox.php

```diff
@@ -160,6 +160,11 @@ public function toArray(): array
         if (!empty($this->expirationAfterHours)) {
             $result['expirationAfterHours'] = $this->expirationAfterHours;
         }
+
+        if ($this->active !== null) {
+            $result['actrive'] = $this->active;
+        }
```

Comment:
So, if active is `null` it will not be part of the response?

also, pozor typo

Reply:
Aah, typo fixed. Hmm, the main thing is not to send it to API (because it would be set to false if omitted). But I think that in php code we shouldn't work with the `toArray`, it's mainly for sending to API? (Otherwise we have `getActive`)